cmd/ocm-backplane/managedJob/createManagedJob.go

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright © 2021 Red Hat, Inc
```

Comment:
```suggestion
Copyright © 2023 Red Hat, Inc
```

Reply:
removed the code header since it's not consistence with other files